.harness/update_env_version.yaml

```diff
@@ -75,3 +75,4 @@ pipeline:
       description: ""
       required: true
       value: <+input>
+  description: please add description
```

Comment:
@NolanJMcCafferty @G-Despacito 
Could you please add pipeline description.
I suggest to specify: what pipeline does, when it is triggered.